README.md

```diff
@@ -59,7 +59,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+the first line declares variable called weather. the second line change the value of weather to "sunny". the third line evaluates to true because it weather and sunny have the same value and type using strict equality.
```

Comment:
This is correct, but each of these lines also evaluates to a value. The first line evaluates to `undefined`. The second evaluates to `"sunny"`, because `=` evaluates to the value to its right. The third, as you said, evaluates to `true`.